sql/src/test/java/io/crate/jobs/PageDownstreamContextTest.java

```diff
@@ -186,4 +187,49 @@ public void testSetBucketOnAKilledCtxReleasesListener() throws Exception {
         // Must not timeout
         listenerReleased.get(1, TimeUnit.SECONDS);
     }
+
+    @Test
+    public void testNonSequentialBucketIds() throws Exception {
+        TestingRowConsumer batchConsumer = new TestingRowConsumer();
+        PageDownstreamContext ctx = getPageDownstreamContext(
+            batchConsumer,
+            PassThroughPagingIterator.oneShot(),
+            3
+        );
+
+        final PageResultListener mockListener = mock(PageResultListener.class);
+
+        Bucket b1 = new CollectionBucket(Collections.singletonList(new Object[] { "foo" }));
+        ctx.setBucket(0, b1, true, mockListener);
+
```

Comment:
empty line

Reply:
This lines serves a purpose. It separates from the second bucket. I just find it more readable.

Reply:
it has 2 empty lines but of course do as you prefer